src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/controllers/PostSubmitCallbackController.java

```diff
@@ -35,34 +35,52 @@
 
     private static final org.slf4j.Logger LOG = getLogger(PostSubmitCallbackController.class);
 
-    private final Deserializer<Callback<AsylumCase>> callbackDeserializer;
     private final PostSubmitCallbackDispatcher<AsylumCase> callbackDispatcher;
 
     public PostSubmitCallbackController(
-        Deserializer<Callback<AsylumCase>> callbackDeserializer,
         PostSubmitCallbackDispatcher<AsylumCase> callbackDispatcher
     ) {
-        requireNonNull(callbackDeserializer, "callbackDeserializer must not be null");
         requireNonNull(callbackDispatcher, "callbackDispatcher must not be null");
 
-        this.callbackDeserializer = callbackDeserializer;
         this.callbackDispatcher = callbackDispatcher;
     }
 
-    @ApiOperation("Handles 'SubmittedEvent' callbacks from CCD")
+    @ApiOperation(
+        value = "Handles 'SubmittedEvent' callbacks from CCD",
+        response = PostSubmitCallbackResponse.class,
+        authorizations = @Authorization(value = "Authorization")
```

Comment:
Is this `authorizations` surplus to requirements?

Reply:
I've changed the authorization headers a little.  
I have put them on the operations themselves rather than api level and added some config which means you can input the Authorization and ServiceAuthorization properly.  So for instance when swagger generates an example curl request you can see the required headers.